stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelStateLogicImpl.kt

```diff
@@ -346,12 +346,19 @@ internal class ChannelStateLogicImpl(
     /**
      * Deletes a member. Doesn't delete in the database.
      *
-     * @param userId Id of the user.
+     * @param member The member to be removed.
      */
-    override fun deleteMember(userId: String) {
-        mutableState.rawMembers = mutableState.rawMembers - userId
+    override fun deleteMember(member: Member) {
+        val user = member.user
+
+        mutableState.rawMembers = mutableState.rawMembers - user.id
 
         mutableState.setMembersCount(mutableState.membersCount.value - 1)
```

Comment:
Why do we have a separate count for members?
Shouldn't we delegate it to the size of the collection?

Reply:
There can be 10 loaded members for a channel with 100 members, so we should have a separate counter.